_projects/2022/sunpy/helioviewer.md

```diff
@@ -0,0 +1,61 @@
+---
+name: HelioViewer Python API Wrapper
+desc: Creating a full Python wrapper for the Unirest Helioviewer API
+requirements:
+ - Familiarity with Python
+ - Some familiarity with Unirest/HTTP requests (ideally)
+difficulty: medium
+issues:
+ - https://github.com/sunpy/sunpy/issues/2860
+ - https://github.com/sunpy/sunpy/issues/2762
+ - https://github.com/sunpy/sunpy/issues/2878
+mentors:
+ - wafels
+ - nabobalis
+ - Helioviewer-Kirill
```

Comment:
@wafels would Helioviewer-Kirill still be up for this or is there someone else I should add?

Reply:
Myself and dgarciabriseno.

Reply:
Helioviewer-Kirill no longer works for Helioviewer.

Reply:
This has been changed now on https://github.com/OpenAstronomy/openastronomy.github.io/pull/302